nmrglue/fileio/varian.py

```diff
@@ -1737,7 +1737,7 @@ def find_shape(pdic):
     # information on imaging experiments
 
     try:
-        shape = [int(pdic["np"]["values"][0]) // 2]
```

Comment:
The double division (//) is required here so that integer division is performed and the elements of shape are intergers.  Another option is to explicitly cast  the elements to integers:

shape = [int(int(pdic["np"]["values"][0]) / 2)]